Assets/Scripts/UI/InGame/SpawnMenu.cs

```diff
@@ -87,10 +87,14 @@ private static bool isPlayerAllDead2()
             return (num == num2);
         }
 
-        public void Update()
+        private void OnEnable()
         {
-            Cursor.visible = true;
-            Screen.lockCursor = false;
+            MenuManager.RegisterOpened();
```

Comment:
I'm not sure about repeating this in all menus. I like that I could put it into `UiNavigationElement`. Maybe we could avoid this code duplication in menus.

Reply:
Sounds good